lib/cli/src/generators/WEB-COMPONENTS/template/stories/1-Button.stories.js

```diff
@@ -0,0 +1,42 @@
+import { html } from 'lit-html';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Button',
+};
+
+export const Text = () => html`
+  <button @click=${action('clicked')}>
+    Hello Button
+  </button>
+`;
+
+export const Emoji = () => html`
+  <button @click=${action('clicked')}>
+    😀 😎 👍 💯
+  </button>
+`;
+
+Emoji.story = {
+  parameters: { notes: 'My notes on a button with emojis' },
```

Comment:
I think we should get rid of `notes` for every framework. It's deprecated or was even removed already.
@ndelangen what do you think?

Reply:
You're right @yannbf, `notes` has been moved to the repo `storybookjs/deprecated-addons` so no 6.0 version will be released.

Reply:
Agree we should get rid of `notes` and pick a better example. Maybe `backgrounds`?

Reply:
I agree, we should leave it out for now.